src/gui/tray/ActivityItem.qml

```diff
@@ -1,264 +1,123 @@
 
-    ToolTip.visible: containsMouse && displayLocation !== ""
-    ToolTip.delay: Qt.styleHints.mousePressAndHoldInterval
-    ToolTip.text: qsTr("In %1").arg(displayLocation)
-        
-    RowLayout {
-        id: activityItem
-        
-        readonly property variant links: model.links
-        
-        readonly property int itemIndex: model.index
-        
-        width: activityMouseArea.width
-        height: Style.trayWindowHeaderHeight
+    ColumnLayout {
+        anchors.left: root.left
+        anchors.right: root.right
+        anchors.leftMargin: 15
+        anchors.rightMargin: 10
+
         spacing: 0
-        
-        Accessible.role: Accessible.ListItem
-        Accessible.name: path !== "" ? qsTr("Open %1 locally").arg(displayPath)
-                                     : message
-        Accessible.onPressAction: activityMouseArea.clicked()   
-        
-        Image {
-            id: activityIcon
-            Layout.alignment: Qt.AlignVCenter | Qt.AlignHCenter
-            Layout.leftMargin: 20
-            Layout.preferredWidth: shareButton.icon.width
-            Layout.preferredHeight: shareButton.icon.height
-            verticalAlignment: Qt.AlignCenter
-            cache: true
-            source: icon
-            sourceSize.height: 64
-            sourceSize.width: 64
-        }
-        
-        Column {
-            id: activityTextColumn
-            Layout.leftMargin: 14
-            Layout.topMargin: 4
-            Layout.bottomMargin: 4
-            Layout.fillWidth: true
-            spacing: 4
-            Layout.alignment: Qt.AlignLeft | Qt.AlignVCenter
-            
-            Text {
-                id: activityTextTitle
-                text: (type === "Activity" || type === "Notification") ? subject : message
-                width: parent.width
-                elide: Text.ElideRight
-                font.pixelSize: Style.topLinePixelSize
-                color: activityTextTitleColor
-            }
-            
-            Text {
-                id: activityTextInfo
-                text: (type === "Sync") ? displayPath
-                                        : (type === "File") ? subject
-                                                            : (type === "Notification") ? message
-                                                                                        : ""
-                height: (text === "") ? 0 : activityTextTitle.height
-                width: parent.width
-                elide: Text.ElideRight
-                font.pixelSize: Style.subLinePixelSize
+
+        Button {
+            id: dismissActionButton
+
+            Layout.preferredWidth: 24
+            Layout.preferredHeight: 24
+            Layout.alignment: Qt.AlignRight
+
+            visible: model.links.length > 0 && linksForActionButtons.length === 0
+
+            background: Rectangle {
+                color: parent.hovered ? "white" : activityHover.color
+                anchors.fill: parent
```

Comment:
```suggestion
```

Reply:
@CarlSchwan Nah, this doesn't work. The Rectangle is now not visible with this change.